lib/renderer.coffee

```diff
@@ -9,7 +9,7 @@ pandocHelper = null # Defer until used
 markdownIt = null # Defer until used
 {scopeForFenceName} = require './extension-helper'
 mathjaxHelper = require './mathjax-helper'
-pathWatcher = require 'pathwatcher-without-runas'
+pathWatcher = require atom.packages.resourcePath + '/node_modules/pathwatcher/lib/main'
```

Comment:
Thanks so much for testing it out yourself @leipert. I now can't get it to work either I must have been doing something odd. Sorry for leading you on a wild goose chase.